content/cif-virtual-catalog/src/main/content/jcr_root/libs/commerce/components/ccifrootfolder/_cq_dialog/.content.xml

```diff
@@ -55,6 +55,10 @@
                                             name="./cq:catalogIdentifier"
                                             renderReadOnly="{Boolean}true"
                                             required="{Boolean}false"/>
+                                    <customfields
+                                            jcr:primaryType="nt:unstructured"
+                                            sling:resourceType="commerce/components/ccifrootfolder/include"
```

Comment:
We should not use "ccif" any more for new things we introduce. I know it is still there for old code, but please not for new.

Reply:
I agree with not using ccif. The situation here is that the new include component belongs to the ccifrootfolder and so putting it under ccifrootfolder looks like a logical choice.

How about renaming the two components starting with ccif to start with cif?

Reply:
> How about renaming the two components starting with ccif to start with cif?
+1 for clean that up if we touching that area anyway

Reply:
@mhaack, I had a quick look at the cleanup which will affect quite a bit more files and I think I would do that refactoring in a dedicated issue to keep the current feature-work clean, so I created CIF-1027.

Reply:
Ok